README.md

```diff
@@ -106,6 +120,18 @@ in your terminal. The project will then be compiled and tested before finally be
 
 We recommend enabling annotation processing in your IDE https://immutables.github.io/apt.html.
 
+## Sinch REST SMS API 1.1.6 to 1.1.7 Migration guide
```

Comment:
Should we write it as 1.x to 2.x migration guide? I think I've seen both variants out in the world and I don't have a strong preference

Reply:
Yes, Spring uses 1.x to 2.x, so we could do the same